man/man5/zfs-events.5

```diff
@@ -693,10 +693,9 @@ full list of all the I/O stages.
 \fBzio_delay\fR
 .ad
 .RS 12n
-The time in ticks (HZ) required for the block layer to service the I/O.  Unlike
+The time in nanoseconds required for the block layer to service the I/O.  Unlike
 \fBzio_delta\fR this does not include any vdev queuing time and is therefore
-solely a measure of the block layer performance.  On most modern Linux systems
-HZ is defined as 1000 making a tick equivalent to 1 millisecond.
+solely a measure of the block layer performance.
```

Comment:
Do I understand correctly that you are actually not changing the meaning of the `zio_delay` payload, becuase it was in nanoseconds before, but the documentation was wrong so you're updating it?

Reply:
That's correct.  See comment: https://github.com/zfsonlinux/zfs/pull/7756#pullrequestreview-150775379

Reply:
I'm being thrown off by the word `required`, I think because there's no concept of a requirement here.  I think this would be more clear as something like `The time elapsed (in nanoseconds) waiting for the block layer to complete the I/O.`